orbit_predictor/predictors/base.py

```diff
@@ -279,6 +281,76 @@ def get_next_pass(self, location, when_utc=None, max_elevation_gt=5,
         else:
             raise NotReachable('Propagation limit date exceeded')
 
+    def eclipses_since(self, when_utc, limit_date=None):
```

Comment:
Perhaps we can mimic the name of `get_next_pass` and call this `get_next_eclipses`? Even though `get_next_passes` and `get_next_eclipses` would be more appropriate.

Reply:
Also, let's have a `when_utc=None` by default which uses `utcnow()`

Reply:
Good suggestions, adding the changes.

Reply:
You're totally right, `eclipses_since` is equivalent to `passes_over`, not `get_next_pass`. Let's keep it that way then.